ppdet/modeling/losses/yolo_loss.py

```diff
@@ -104,15 +109,17 @@ def _get_fine_grained_loss(self, outputs, targets, gt_box, batch_size,
             "YOLOv3 output layer number not equal target number"
 
         downsample = 32
-        loss_xys, loss_whs, loss_objs, loss_clss = [], [], [], []
+        loss_xys, loss_whs, loss_locs, loss_objs, loss_clss = [], [], [], [], []
         for i, (output, target,
                 anchors) in enumerate(zip(outputs, targets, mask_anchors)):
             an_num = len(anchors) // 2
             x, y, w, h, obj, cls = self._split_output(output, an_num,
                                                       num_classes)
             tx, ty, tw, th, tscale, tobj, tcls = self._split_target(target)
+            loss_giou = self._IoUloss(x, y, w, h, tx, ty, tw, th, anchors, downsample)
```

Comment:
Whether need iou-loss should based the setting in the config

Reply:
It can not be configured in the config, I can modify the code to support it